developer-resources/Developer-Guidelines.md

```diff
@@ -162,6 +162,15 @@ code is committed into git.
 
 * For java unit tests use /src/test/java folder of the module (standard maven location), and postfix java file names with "Test".
 
+### Working with Dates
+
+Egeria manages date/time instances as absolute Unix epoch timestamp and always stores them as `java.lang.Long`.
+Different representation can be used across layers but is recommended to follow the Egeria best practice:
```

Comment:
In other Egeria APIs that might be developed, it is strongly recommended to store dates and times as a Long or Date. It is possible to expose additional localised date representations if required.

Reply:
perhaps it should say 'It is possible to expose localized dates if required;? The 'additional' almost implies the current is localized?

Reply:
I think the addition piece is important - as it implied you should always have the unlocalised version.  How about "In addition, it is possible to expose localised date representations if required."